File: app/src/test/java/me/sheimi/android/utils/CodeGuesserTest.java
package me.sheimi.android.utils;

import static org.junit.Assert.assertEquals;

/**
 *
 */
public class CodeGuesserTest {

    @org.junit.Test
    public void testGuessCodeType() {
        assertEquals("expect to recognise java files", "text/x-java", CodeGuesser.guessCodeType("test.java"));
    }
}

File: app/src/test/java/me/sheimi/android/utils/CodeGuesserTest.kt
package me.sheimi.android.utils

import org.junit.Assert.assertEquals
import org.junit.Test

/**
 *
 */
class CodeGuesserTest {

    @Test
    fun testGuessCodeType() {
        assertEquals("expect to recognise java files", "text/x-java", CodeGuesser.guessCodeType("test.java"))
    }
}
